documentation/customisation.md

```diff
@@ -0,0 +1,139 @@
+- etc.
+
+The process to customise a docker image for your needs is described below
+
+## XL Deploy
+
+When creating a Dockerfile with custom resources added, always remember that the owning user:group combination _must_ be `10001:0`, or else XL Deploy will not be able to read the files. 
+
+To start off with, create a `Dockerfile` that resembles the following. 
+
+Please note that certain JARS will go into certain paths. You shouldn't add a Oracle JAR to the `ext/` folder, for example. If you are unsure where a JAR should be added, please get in touch with our support team. 
+
+NOTE: `${APP_HOME}` points to the path `/opt/xebialabs/xl-deploy-server` by default
+
+```docker
+FROM xebialabs/xl-deploy:9.5.0
+
+###################################################################################
+# PLUGINS                                                                         #
+# Plugins should be placed under ${APP_HOME}/default-plugins/ #
+###################################################################################
+
+COPY --chown=10001:0 files/xld-liquibase-plugin-5.0.1.xldp /opt/xebialabs/xl-deploy-server/default-plugins/
+
+# Add plugin from url
+ADD --chown=10001:0 https://dist.xebialabs.com/public/community/xl-deploy/command2-plugin/3.9.1-1/command2-plugin-3.9.1-1.jar /opt/xebialabs/xl-deploy-server/default-plugins/
+
+##########################################################################
+# EXTENSIONS                                                             #
+# Extensions should be placed under ${APP_HOME}/ext
+##########################################################################
+ADD --chown=10001:0 files/ext /opt/xebialabs/xl-deploy-server/ext/
+
+##########################################################################
+# HOTFIXES                                                               #
+##########################################################################
+ADD --chown=10001:0 files/lib-hotfix.jar /opt/xebialabs/xl-deploy-server/hotfix/lib/
+ADD --chown=10001:0 files/plugin-hotfix.jar /opt/xebialabs/xl-deploy-server/hotfix/plugin/
+ADD --chown=10001:0 files/sattelite-lib-hotfix.jar /opt/xebialabs/xl-deploy-server/hotfix/sattelite-lib/
+
+##########################################################################
+# LIBRARIES                                                              #
+##########################################################################
+ADD --chown=10001:0 files/ojdbc6.jar /opt/xebialabs/xl-deploy-server/lib/
+```
+
+Please note that there are seperate hotfix directories where you can place hotfix JARs, depending on the type of hotfix. Please consult our support team in case you are uncertain. 
+
+For an overview of how `ADD` and `COPY` works, please refer to [the documentation](https://docs.docker.com/engine/reference/builder/#add)
+
+Once you are satisfied with your Dockerfile, run the following command in the same directory
+
+`docker build -t xl-deploy-custom:9.5.0 .`
```

Comment:
- change this to create :9.0 tag based on above comment.

Reply:
no need to change